apps/studio/sanity.config.ts

```diff
@@ -44,30 +44,12 @@ const sharedSettings = definePlugin({
   },
 })
 
-// If we're on a preview deployment we'll want the iframe URLs to point to the same preview deployment
-function maybeGitBranchUrl(url: string) {
```

Comment:
This stuff was always very brittle.
Moving forward deployments will always point to the production URLs.
If a PR has changes that require the Visual Editing Studio to open deployments of Nuxt/Next/Etc from the same branch, that is done by adding environment variables for that branch on the env settings: https://vercel.com/sanity-sandbox/visual-editing-studio/settings/environment-variables

Reply:
Will that be a manual process per branch?

I guess this will be a requirement for almost every PR unless I'm mistaken?

Reply:
Yes, but not for long, I'll make a github action that automatically sets them 🙌